be/src/vec/functions/match.h

```diff
@@ -56,6 +56,8 @@ const std::string MATCH_PHRASE_FUNCTION = "match_phrase";
 
 class FunctionMatchBase : public IFunction {
 public:
+    bool use_default_implementation_for_nulls() const override { return false; }
```

Comment:
Unused?

Reply:
do not use default impl for nulls, match null inside match function